khanh19934/backend-coding-test
# Feat/make api document

## What problem does this PR solve

This PR is for making documentation for API of this application.

## How does this PR solve it

- Write documentation for this API in readme
- Setup swagger for API doc

## How can reviewers test the changes

- Check the document on readme.
- Start application and navigate to `http://localhost:8010/api-docs` to check the api document. The api doc should be like this 
<img width="1782" alt="Screen Shot 2021-08-19 at 20 48 04" src="https://user-images.githubusercontent.com/18601105/130080027-224ee3bc-1525-46db-b07a-177a426aa975.png">


## What Documentation/References will help understand the problem and solution

N/A

## Any limitation or related work yet to be done

N/A
